packages/ra-ui-materialui/src/input/BooleanInput.spec.tsx

```diff
@@ -90,10 +92,10 @@ describe('<BooleanInput />', () => {
     });
 
     it('should be checked if the value is true and initialValue is false', () => {
-        const { getByLabelText } = render(
-            <Form
+        const { getByLabelText } = renderWithRedux(
```

Comment:
Shouldn't most tests that use react-final-form's `Form` with `initialValues` be change like this one?

Reply:
You're right

Reply:
Not all of them though. Only those where we use both the initialValues of `Form` and initialValue of the field

Reply:
So, that was actually the only test where we needed that

Reply:
👍